src/components/DashboardEditor/DashboardEditorHeader/DashboardEditorHeader.jsx

```diff
@@ -200,15 +201,16 @@ const DashboardEditorHeader = ({
           />
         )}
         {onCancel && (
-          <Button kind="tertiary" size="field" onClick={onCancel}>
+          <Button kind="secondary" size="field" onClick={onCancel}>
             {mergedI18n.headerCancelButton}
           </Button>
         )}
         {onSubmit && (
           <Button
             size="field"
-            disabled={submitDisabled}
-            onClick={() => onSubmit(dashboardJson)}>
+            disabled={isSubmitDisabled}
+            onClick={() => onSubmit(dashboardJson, setIsSubmitLoading)}
```

Comment:
Interesting, why a `setIsSubmitLoading` callback to manage an internal state instead of an `isLoading` prop like the `isSubmitDisabled`? I would choose prop over state, but perhaps there is a reason I don't see.

Reply:
I was thinking that the `isLoading` is directly related to `onSubmit`, but I see the we only use the prop in other examples like ComposedModal. I'll swap this to a prop instead to stay consistent

Reply:
Swapped to the `isSubmitLoading` prop